### PR TITLE
Core, wb techdraw, wb part :Removed deprecated func

### PR DESCRIPTION
**Pull request by mosfet80**
QGIPrimPath::setHighlighted, TopoShape::setShapePlacement and TopoShape::getShapePlacement are obsolete and never used

### CODE REVIEW

**Comment by Pesc0** [comment]
@JohnAD @chennes check toposhape changes. I think It's fine to remove these, they seem duplicated anyway.

File: src/Gui/NavigationStyle.cpp [hunk 1]
```diff
@@ -1020,8 +1020,7 @@ void NavigationStyle::moveCursorPosition()
 
 void NavigationStyle::redraw()
 {
-    if (mouseSelection)
-        mouseSelection->redraw();
```
Comment (chennes) [inline src/Gui/NavigationStyle.cpp:1024]: What's going on with this removal?

Comment (wwmayer) [reply src/Gui/NavigationStyle.cpp:1024]: With AbstractMouseSelection::redraw() also NavigationStyle::redraw() can be completely removed.

**Comment by chennes** [comment]
@mosfet80 Can you check over this again in light of @wwmayer's comment above?

**Comment by sliptonic** [comment]
Moving back to draft.  @mosfet80  Please reset when ready to review.

**Pull request merged**